recipes/example/meta.yaml

```diff
@@ -1,81 +0,0 @@
-# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```

Comment:
Please do not touch this file, could you please revert it?

Reply:
Done.